src/CMakeLists.txt

```diff
@@ -229,3 +231,36 @@ IF(UNIX AND NOT APPLE)
     INSTALL(DIRECTORY ${CMAKE_SOURCE_DIR}/docs/man/ DESTINATION share/man)
 ENDIF(UNIX AND NOT APPLE)
 
+# add install target:
+# WINDOWS and MINGW-w64
+IF(WIN32 AND CMAKE_COMPILER_IS_MINGW)
+    IF("${CMAKE_INSTALL_PREFIX}" STREQUAL "")
```

Comment:
Here is what I added on the second commit.

Windows CMAKE defaults CMAKE_INSTALL_PREFIX to `C:\Program Files\${PROJECT_NAME}` which luckily (?) needs admin privileges to be messed up...

BUT if you leave the variable DEFINED but empty then it will usually default to the root of the drive you'r running on your shell/process... took me a while to figure where cmak did put those files...

This check prevents this and defaults in a specific subfolder of CMAKE project binary root dir, where it's unlikely any problem may arise (and files are handy to be copypasted to a location of your choice, I guess).